examples/jsm/loaders/DRACOLoader.js

```diff
@@ -209,6 +209,8 @@ class DRACOLoader extends Loader {
 
 				this._assignVertexColorSpace( attribute, result.vertexColorSpace );
 
+				attribute.normalized = true;
```

Comment:
This would need to be conditional on the type of data — float32 attributes cannot be normalized. Possibly something like this?

```suggestion
				attribute.normalized = ( array instanceof Float32Array ) === false;
```

Reply:
good shout